deploy/playbooks/roles/common/templates/prod.py.j2

```diff
@@ -84,6 +84,29 @@ polling_cycle = 120
 
 {% if purge_repos is defined %}
 purge_repos = {{ purge_repos }}
+
+purge_rotation = {
+    'ceph': {
```

Comment:
Let's add ``kernel`` here as well, we'll need to ensure the ``testing`` ref stays around.